icepyx/core/query.py

```diff
@@ -908,7 +908,7 @@ def is_ec2():
         self._email = email
 
     # DevGoal: check to make sure the see also bits of the docstrings work properly in RTD
-    def avail_granules(self, ids=False, cycles=False, tracks=False, s3urls=False):
+    def avail_granules(self, ids=False, cycles=False, tracks=False, cloud=False):
```

Comment:
@tsutterley @jpswinski Would this be a breaking change for SlideRule? My aim here was to make it more generic (since it's needed for authentication as well as directing data access), but there's no reason the keyword can't stay as `s3urls` if it prevents issues downstream.